src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -50,11 +51,48 @@
     (normalize-input current v)
     current))
 
+(defn- estimated-fees
+  [{:keys [loading-suggested-routes? fees native-currency-symbol amount receiver]}]
+  [rn/view {:style style/estimated-fees-container}
+   [rn/view {:style style/estimated-fees-content-container}
+    [quo/button
+     {:icon-only?          true
+      :type                :outline
+      :size                32
+      :inner-style         {:opacity 1}
```

Comment:
This is because when in loading state the button is disabled which leads the button to have an opacity of 0.3, so I need to force opacity to be 1 on `inner-style` to avoid this behavior wether if the button is disabled or enabled.

Reply:
So we want that the button looks enabled even when it's disabled?

Is it too bad to show it as disabled? or just trigger nothing when the routes are being loaded?

Just thinking if this needs to be addressed in a more general way if we face it again.